data_model/derive/src/partially_tagged.rs

```diff
@@ -223,12 +232,64 @@ pub fn impl_partially_tagged_deserialize(enum_: &PartiallyTaggedEnum) -> TokenSt
                     Other(#internal_repr_ident),
                 }
 
-                let wrapper = DeserializeHelper::deserialize(deserializer)?;
+                // TODO: remove once `serde::__private::ContentDeserializer` properly handles `u128`.
+                // Tracking issue: https://github.com/serde-rs/serde/issues/2230
+                impl<'de> ::serde::Deserialize<'de> for #deser_helper {
+                    fn deserialize<D: ::serde::Deserializer<'de>>(
+                        deserializer: D,
+                    ) -> Result<Self, D::Error> {
+                        #[cfg(feature = "std")]
+                        let mut errors = Vec::new();
+
+                        let content = serde_json::Value::deserialize(deserializer)?;
+                        #(
+                            {
+                                // FIXME: Elide expensive clone.
```

Comment:
`&'de Value` implement [`Deserializer<'de>`](https://docs.rs/serde_json/latest/serde_json/enum.Value.html#impl-Deserializer%3C'de%3E-for-%26'de+Value) isn't it possible to use `&content` instead of `clone`?

Reply:
Indeed. Fixing now.